src/tests/ftest/io/nvme_pool_capacity.yaml

```diff
@@ -0,0 +1,48 @@
+server_manager:
+    srv_timeout: 500
+hosts:
+  test_servers:
```

Comment:
It would be better if we can make the indentations consistent.

Reply:
Will fix it...Thanks.